.github/workflows/deploy-to-beta-manually.yml

```diff
@@ -0,0 +1,95 @@
+name: Deploy to beta manually
+
+concurrency:
+  group: deploy-beta
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
```

Comment:
Regarding to the issue https://github.com/blumilksoftware/keating/issues/44,, workflow should be triggered on merge to the main branch.

See
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs